CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+v1.8.0-RC1 / 2020-03-06
```

Comment:
update this with: 
- 1.8.0-RC2  (include changes that went only in 1.8.0-RC2)
- 1.8.0 (all changes from 1.8.0 RC1 and RC2 + additional fix that was only in 1.8.0)

Reply:
Done